src/classes/CAO_Constants_API.cls

```diff
@@ -0,0 +1,96 @@
+/*
+    Copyright (c) 2018, Salesforce.org
+    All rights reserved.
+    
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+    
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+* @author Salesforce.org
+* @date 2018
+* @group API
+* @group-content ../../ApexDocContent/API.htm
+* @description API class that exposes CAO (Contacts and Organizations) Constants and helper methods.
+*/
+global class CAO_Constants_API {
+
+    /*******************************************************************************************************
+    * @description One-to-One value for system processor field
+    */ 
+    global static final String ONE_TO_ONE_PROCESSOR {
+        get {return CAO_Constants.ONE_TO_ONE_PROCESSOR;}
```

Comment:
@lparrott I ran into an apex recently bug where not having a `set;` would cause a compile error when changing api versions. The recommended fix was to include `private set;` anywhere where a set was implied but not specified.

Reply:
But since we are not changing API versions (the whole class is new) maybe this will work? I'd just hate to add cruft unnecessarily...

Reply:
It's possible that changing from 43 to a newer version will introduce this bug again. Plus, in order to complete the v43 changes I had to fix every single method that had a getter without a setter, so it would be better if anything new stays consistent with that pattern.

Reply:
That's an interesting bug! I've added the private set methods just in case.

Reply:
Thanks for making that change.